src/LanguageServerProtocol.fs

```diff
@@ -848,14 +848,36 @@ module Types =
         /// Experimental client capabilities.
         Experimental: JToken option
     }
+    
+    type WorkspaceFolder = {
+        /// The associated URI for this workspace folder.
+        Uri: DocumentUri;
+
+        /// The name of the workspace folder. Defaults to the
+        /// uri's basename.
+        Name: string;
+    }
+    
+    type ClientInfo = {
+        Name: string
+        Version: string option
+    }
 
     type InitializeParams = {
         ProcessId: int option
+        /// Information about the client.
+        /// @since 3.15.0
+        ClientInfo: ClientInfo option
```

Comment:
I don't know why, but CI thinks this field is duplicated on this structure.

Reply:
Hm. Maybe that's because my branch is forked off 0.3.1. Let me rebase on master and try again.

Reply:
Yeah, so between 0.3.1 and current main `ClientInfo` was added as a `JToken option`. I've removed the dupe favouring a properly typed version.